### ISSUE TITLE
[bug] ENS IPFS avatars cannot be loaded

### ISSUE THREAD

**Issue by sprice** [tmm/wagmi#59]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Package Version

0.1.2

### Current Behavior

Using the example code from https://wagmi-xyz.vercel.app/guides/connect-wallet, I am connecting to my Ethereum wallet which has an [ENS entry](https://app.ens.domains/name/shawnprice.eth/details)

The avatar does not display and the console reads `Fetch API cannot load ipfs://QmPMc4tcBsMqLRuCQtPmPe84bpSjrC3Ky7t3JWuHXYB4aS/8929. URL scheme "ipfs" is not supported.`

### Expected Behavior

That my ENS avatar displays correctly

### Steps To Reproduce

Run the example code from the `wagmi` docs (https://wagmi-xyz.vercel.app/guides/connect-wallet) and connect to an Ethereum address that has an ENS entry with an avatar set according to [these docs](https://medium.com/the-ethereum-name-service/step-by-step-guide-to-setting-an-nft-as-your-ens-profile-avatar-3562d39567fc)

### Anything else?

This package is amazing! Thank you.

**Comment by sprice** [comment]
I just realized i can connect to Metamask on the https://wagmi-xyz.vercel.app/guides/connect-wallet page and the same error occurs in the Console.

**Comment by 0xOlias** [comment]
The [ENS avatar field format spec](https://gist.github.com/Arachnid/9db60bd75277969ee1689c8742b75182) supports several URI types, most of which are not compatible with `<img>` tags out-of-the-box. If you're looking for a quick and effective solution, take a look at the [`davatar`](https://www.npmjs.com/package/@davatar/react) library.

I think the wagmi docs should probably remove the ens avatar `<img>` tag from the guide, because it will likely confuse more devs. wagmi could also consider offering a `<img>`-ready URI by re-implementing some of the functionality of `davatar`, but I'm not sure if that's aligned with wagmi's goals.

**Comment by peduarte** [comment]
semiwhat related: https://github.com/tmm/wagmi/discussions/33

**Comment by Omar-Aziz** [comment]
try something like this

```JavaScript
import {  useProvider } from 'wagmi';
...
  const provider = useProvider();
  const getEnsAvatar = async (nameOrAddress: string) => {
    const avatarUrl = await provider.getAvatar(nameOrAddress);
    return avatarUrl;
  }
...
```

**Comment by tmm** [comment]
Fixed as of `ethers@5.5.4` https://github.com/ethers-io/ethers.js/pull/2271#issuecomment-1025334108
